docs/src/main/paradox/docs/delta/api/supervision-api.md

```diff
@@ -0,0 +1,19 @@
+# Supervision
+
+This endpoint returns information about the projection supervisor.
```

Comment:
This endpoint returns information about the projections running on the current node.